yaas/host.py

```diff
@@ -1,4 +1,56 @@
 # coding: utf-8
 
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+import pprint
+import requests
+import sys
+
+from . import config
+
+
 def command(parser, args):
-    pass
+    """ Interact with registered agents. """
+    subparsers = parser.add_subparsers(dest='subcommand')
+    subcommands = {
+        'list': _list,
+        }
+    for name, fn in subcommands.items():
+        subparsers.add_parser(name, help=inspect.getdoc(fn))
+    subargs, extra = parser.parse_known_args(args)
+    subcommands[subargs.subcommand](
+        subparsers.choices[subargs.subcommand],
+        extra)
+
+
+def _list(parser, args):
+    """ List all registered agents. """
```

Comment:
This should be registered hosts. If you use the server to deploy agents, it's almost an implementation detail that the server uses agents. The endpoint is called hosts, not agents.